pkg/systemd/overview-cards/usageCard.jsx

```diff
@@ -147,7 +147,7 @@ export class UsageCard extends React.Component {
                                     <Progress value={this.state.memUsed}
                                         className="pf-m-sm"
                                         min={0} max={this.state.memTotal}
-                                        variant={fraction > 0.9 ? ProgressVariant.danger : null}
+                                        variant={fraction > 0.9 ? ProgressVariant.danger : undefined}
```

Comment:
What is the reason for this change?

Reply:
Unrelated but it's a functionally the same. Patternfly takes in either variant or undefined type. Null isn't an option since it's an optional prop

Reply:
Split it to another commit